aptos-move/move-examples/feature-sandbox/sources/sandbox_messaging.move

```diff
@@ -0,0 +1,213 @@
+/// An example module similar to the hello blockchain demo
+///
+/// This module is to show off all features that can be used in Move in a simple way.
+///
+/// Here is an example of a doc comment.  The doc comments will be used when running the doc generator,
+/// and at this location it will document for the module.
+module feature_sandbox::sandbox_messaging {
+    // Imports can be used as an individual function
+    use std::error::not_found;
+    use std::error::permission_denied;
+
+    // Imports can be used for structs
+    use std::string::String;
+
+    // Imports can also be used for modules
+    use std::signer;
+    use aptos_framework::account;
+    use aptos_framework::event;
+
+    // Imports can be test only
+    #[test_only]
+    use std::string::utf8;
+
+    // Errors allow for human readable messages associated with an abort in Move.  The message from the
+    // doc comment above the error message is provided to users when an abort with this error occurs.  If
+    // no message is given, only the name of the error will be given.
+    //
+    // By convention, errors start with E to represent an error, and start at 1 and increase.
+    /// There is no message holder present at address
+    const ENO_MESSAGE_HOLDER: u64 = 1;
+    /// The action is an admin action, and the caller is not the admin
+    const ENOT_ADMIN: u64 = 2;
+
+    //:!:>resource
+    /// A message holder resource for this example.  This resource contains
+    /// an individual string as a message, and an event handle for Aptos events.
+    ///
+    /// You can see the generic phantom type for the MessageHolder.  This allows us to declare
```

Comment:
For detailed explanations like this, I think we should just link to the Move book to avoid duplication and this getting out of date

Reply:
I removed the detailed explanations, and cut it down just to feature specific comments for me as a reader.